SAP/Script/FXVProduct.py

```diff
@@ -254,6 +254,11 @@ def air_handling_values(sap_field_values):
   air_handling_configurations["fan_motor_rpm_a"] = sap_field_values["Fan Motor RPM A"].strip()
   air_handling_configurations["fan_motor_type"] = sap_field_values["Fan Motor Type"].strip()
   air_handling_configurations["fan_motor_quantity_main_a"] = sap_field_values["Fan Motor Quantity - Main A:"].strip()
+  air_handling_configurations["fan_motor_rpm__a_pony"] = sap_field_values["Fan Motor RPM A Pony:"].strip()
+  air_handling_configurations["fan_motor_type_pony"] = sap_field_values["Fan Motor Type - Pony:"].strip()
+  air_handling_configurations["fan_motor_quantity_pony_a"] = sap_field_values["Fan Motor Quantity - Pony A:"].strip()
+  air_handling_configurations["fan_motor_options_a_pony:"] = sap_field_values["Fan Motor Options A Pony:"].strip()
```

Comment:
@vaishnavir9501 remove this `:` line 260

Reply:
colon has been removed.